shared/chat/conversations-list/index.desktop.js

```diff
@@ -115,52 +102,71 @@ const BottomLine = ({participantNeedToRekey, youNeedToRekey, isMuted, boldOverri
-      style={{...rowContainerStyle, backgroundColor: rowBackgroundColor(hasUnread, isSelected)}}>
+      onClick={() => props.onSelectConversation(props.conversationIDKey)}
+      style={{...rowContainerStyle, backgroundColor: props.backgroundColor}}
+      title={`${props.unreadCount} unread`}
+    >
       <Avatars
-        hasUnread={hasUnread}
-        isMuted={isMuted}
-        isSelected={isSelected}
-        participantNeedToRekey={participantNeedToRekey}
-        participants={participants}
-        youNeedToRekey={youNeedToRekey} />
-      <div style={{...globalStyles.flexBoxColumn, ...conversationRowStyle, borderBottom: (!isSelected && !hasUnread) ? `solid 1px ${globalColors.black_10}` : 'solid 1px transparent'}}>
+        backgroundColor={props.backgroundColor}
+        hasUnread={props.hasUnread}
+        isMuted={props.isMuted}
+        isSelected={props.isSelected}
+        participantNeedToRekey={props.participantNeedToRekey}
+        participants={props.participants}
+        youNeedToRekey={props.youNeedToRekey}
+      />
+      <div style={{...globalStyles.flexBoxColumn, ...conversationRowStyle, borderBottom: (!props.isSelected && !props.hasUnread) ? `solid 1px ${globalColors.black_10}` : 'solid 1px transparent'}}>
         <TopLine
-          hasUnread={hasUnread}
-          conversation={conversation}
-          boldOverride={boldOverride}
-          participants={participants}
-          nowOverride={nowOverride}
-          subColor={subColor}
-          isSelected={isSelected} />
+          commaColor={props.commaColor}
+          hasUnread={props.hasUnread}
+          participants={props.participants}
+          showBold={props.showBold}
+          subColor={props.subColor}
+          timestamp={props.timestamp}
+          usernameColor={props.usernameColor}
+        />
         <BottomLine
-          participantNeedToRekey={participantNeedToRekey}
-          youNeedToRekey={youNeedToRekey}
-          conversation={conversation}
-          subColor={subColor}
-          boldOverride={boldOverride}
-          isMuted={isMuted} />
+          isMuted={props.isMuted}
+          participantNeedToRekey={props.participantNeedToRekey}
+          showBold={props.showBold}
+          snippet={props.snippet}
+          subColor={props.subColor}
+          youNeedToRekey={props.youNeedToRekey}
+        />
       </div>
     </div>
   )
 }
 
+const Row = shouldUpdate((props: RowProps, nextProps: RowProps) => {
+  const different =
+    props.conversationIDKey !== nextProps.conversationIDKey ||
+    props.unreadCount !== nextProps.unreadCount ||
+    props.isSelected !== nextProps.isSelected ||
+    props.isMuted !== nextProps.isMuted ||
+    props.youNeedToRekey !== nextProps.youNeedToRekey ||
+    props.participantNeedToRekey !== nextProps.participantNeedToRekey ||
+    props.timestamp !== nextProps.timestamp ||
+    props.snippet !== nextProps.snippet ||
+    !props.participants.equals(nextProps.participants)
+  return different
```

Comment:
:sheep: This seems like it could surprise us in the future when we add something and it doesn't get updates. Is that worth the perf gains here? Would a blacklist of props, rather that whitelist, be feasible?